docs/UserGuide.md

```diff
@@ -339,78 +350,160 @@ Memo: NEW_MEMO
 
-- mark /s 2 /c CS2103T
+**Sample Usage**:
 
-**Successful Output:**
+1. Assuming you want to mark the 2nd Student in your class CS2103T as present
+2. Enter the following commands:
+```
+view /c 1
+mark /s 2 /c CS2103T
+```
+3. The result box will display the following message upon success
+```
+You have marked Jamie present!
+```
+
+<box type="info">
 
-- STUDENT_NAME has been marked present!
+**Information:**
 
-**Unsuccessful Output:**
+* Both STUDENT_INDEX and CLASS_NAME are compulsory.
+* STUDENT_INDEX must be a valid positive integer shown in the displayed student list.
+* CLASS_NAME is not case-sensitive.
+* CLASS_NAME must be the name of a class that already exists.
+
+</box>
+
+<box type="warning">
+
+**Caution:**
+
+* The student you want to mark should not already be marked present.
+
+</box>
+
+<box type="tip">
+
+**Tip:**
+
+* If all your students are present, use our `markall` command instead!
+
+</box>
+
+**Successful Output:**
+* STUDENT_NAME has been marked present!
 
-- Index is not a non-zero unsigned integer
-- The Class name (CS9999) you provided does not exist!
-- STUDENT_NAME has already been marked present!
+**Unsuccessful Output:**
+* Index is not a non-zero unsigned integer.
+* The Class name (CS1234) you provided does not exist!
+* STUDENT_NAME has already been marked present!
 
 <br>
 
-### Marking your student's attendance as absent : `unmark`
+### Marking a student absent : `unmark`
 
 **Command Format**: `unmark /s STUDENT_INDEX /c CLASS_NAME`
 
-- Both STUDENT_INDEX and CLASS_NAME are compulsory
-- CLASS_INDEX must be a valid positive integer shown in the displayed student list
-- CLASS_NAME is not case-sensitive
-- CLASS_NAME must be the name of a class that already exists
-- Selected student should have been marked present
-
 **Examples**
+* `unmark /s 2 /c CS2103T` : Marks the 2nd student in your class CS2103T as absent.
+* `unmark /s 4 /c CS2102` : Marks the 4th student in your class CS2103T as absent.
 
-- unmark /s 2 /c CS2103T
+**Sample Usage**
 
-**Successful Output:**
+1. Assuming you want to mark the 2nd student in your class CS2103T.
+2. Enter the following commands:
+```
+view /c 1
+mark /s 2 /c CS2103T
+```
+3. Upon success, the result box will display the following message:
+```
+Jamie has been marked absent!
+```
+
+<box type="info">
 
-- Emily has been marked absent!!
+**Information:**
 
-**Unsuccessful Output:**
+* Both STUDENT_INDEX and CLASS_NAME are compulsory.
+* CLASS_INDEX must be a valid positive integer shown in the displayed student list.
+* CLASS_NAME is not case-sensitive.
+* CLASS_NAME must be the name of a class that already exists.
 
-- Index is not a non-zero unsigned integer
-- The Class name (CS9999) you provided does not exist!
-- Emily has already been marked absent!
+</box>
+
+<box type="warning">
+
+**Caution:**
+
+* The student you want to unmark should not already be marked absent.
+
+</box>
+
+**Successful Output:**
+* Emily has been marked absent!
```

Comment:
Perhaps generalise the parameters as STUDENT_NAME to standardise with other commands!